biletmajster-backend/Controllers/EventApi.cs

```diff
@@ -123,6 +137,7 @@ public class EventApiController : ControllerBase
             {
                 ModelState.Clear();
                 ModelState.AddModelError("", "Something went wrong while savin");
+                _logger.LogDebug($"Event with name: {name} was not added, something went wrong");
```

Comment:
remove `something went wrong` this message is for you not for the api consumer, put there information that will help you identify the problem